index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Maher</title>
+    <link rel="stylesheet" href="css/style.css">
+    <!-- link swiper css -->
+    <link rel="stylesheet" href="CSS/swiper-bundle.min.css">
+    <!-- BOX ICONS -->
+</head>
+<body>
+    <!-- Header -->
+        <!-- nav -->
+        <div class="nav container">
+            <!-- Logo -->
+            <div class="navbar">
+            <p  href="#">Maher</p>
+          </div>
+          <div class="end">
```

Comment:
- [ ] Kindly wrap your toolbar in a `<header>`  tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info